client/coral-embed-stream/src/CommentStream.js

```diff
@@ -90,137 +96,148 @@ class CommentStream extends Component {
-                    currentUser={this.props.auth.user}/>
-                </div>
-                <div className="commentActionsRight">
-                  <FlagButton
-                    addNotification={this.props.addNotification}
-                    id={commentId}
-                    flag={actions[comment.flag]}
-                    postAction={this.props.postAction}
-                    deleteAction={this.props.deleteAction}
-                    addItem={this.props.addItem}
-                    updateItem={this.props.updateItem}
-                    currentUser={this.props.auth.user}/>
-                    <PermalinkButton
-                      comment_id={commentId}
-                      asset_id={comment.asset_id}/>
-                </div>
-                  <ReplyBox
-                    addNotification={this.props.addNotification}
-                    postItem={this.props.postItem}
-                    appendItemArray={this.props.appendItemArray}
-                    updateItem={this.props.updateItem}
-                    id={rootItemId}
-                    author={user}
-                    parent_id={commentId}
-                    premod={this.props.config.moderation}
-                    showReply={comment.showReply}/>
-                  {
-                    comment.children &&
-                    comment.children.map((replyId) => {
-                      let reply = this.props.items.comments[replyId];
-                      return <div className="reply" key={replyId}>
-                        <hr aria-hidden={true}/>
-                        <AuthorName author={users[reply.author_id]}/>
-                        <PubDate created_at={reply.created_at}/>
-                        <Content body={reply.body}/>
-                        <div className="replyActionsLeft">
-                            <ReplyButton
-                              updateItem={this.props.updateItem}
-                              id={replyId}
-                              showReply={reply.showReply}/>
-                            <LikeButton
-                              addNotification={this.props.addNotification}
-                              id={replyId}
-                              like={this.props.items.actions[reply.like]}
-                              postAction={this.props.postAction}
-                              deleteAction={this.props.deleteAction}
-                              addItem={this.props.addItem}
-                              updateItem={this.props.updateItem}
-                              currentUser={this.props.auth.user}/>
-                          </div>
-                          <div className="replyActionsRight">
-                            <FlagButton
+
+          <TabBar onChange={this.props.changeTab} activeTab={activeTab}>
+            <Tab><Count id={rootItemId} items={this.props.items}/></Tab>
+            <Tab>Settings</Tab>
+          </TabBar>
+
+          <ContentBox show={activeTab === 0}>
```

Comment:
could we call this something like `<TabContent>` so we know they belong together? or is this being used somewhere else?